integration/package-operator/image_prefix_override_test.go

```diff
@@ -0,0 +1,90 @@
+//go:build integration
+
+package packageoperator
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	"github.com/go-logr/logr"
+	"github.com/go-logr/logr/testr"
+	"github.com/google/go-containerregistry/pkg/crane"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	corev1alpha1 "package-operator.run/apis/core/v1alpha1"
+	manifestsv1alpha1 "package-operator.run/apis/manifests/v1alpha1"
+)
+
+const (
+	packageName      = "image-prefix-override"
+	packageNamespace = "default"
+)
+
+func TestImagePrefixOverride(t *testing.T) {
+	// Mirror workload image from registry/src to registry/mirror.
+	// Package image is pushed directly to /mirror so no need to mirror it.
+	require.NoError(t, crane.Copy(
+		imageURLFromHost(TestStubImageSrc),
+		imageURLFromHost(TestStubImageMirror),
+	))
```

Comment:
I think it would be better to move this to `cmd/build/cluster.go`, like we talked about. :)